### DISCUSSION TITLE
Opposite way - Arduino to Crossfire TX?

### DISCUSSION THREAD

**Discussion by FPVZaphod** [ZZ-Cat/CRSFforArduino#80] [Ideas]
Hello,
is it possible to add the reversed way to the library? Sending RC Channels to a Crossfire or ELRS Transmitter?
I would like to create a "Software Defined RC Box", where you connect one or two Gaming Joysticks to a Microcontroller, and send the Channel Values to a Crossfire TX. 
thanks,
Michael

**Comment by ZZ-Cat** [comment]
Not in the current version of CRSF for Arduino.  
My library would need to have a way to write out CRSF packets (containing both the RC Channels data and the link statistics data).  
Which is currently not yet implemented.

This would be a very interesting concept to explore, by the way.
If you would like this to be added into the development pipeline, consider submitting an [Issue](https://github.com/ZZ-Cat/CRSFforArduino/issues/new/choose). 'Cause I'd love to experiment with this. =^/.~=
